### DISCUSSION TITLE
Changelogs

### DISCUSSION THREAD

**Discussion by bstro** [adobe/react-spectrum#3337] [Q&A] [answered]
I've often looked for changelogs for react-aria/stately releases. Is there any convenient location to get a birds-eye view of what changes from release to release so I can decide what/when to upgrade?

Thanks!

**Comment by devongovett** [comment, marked as answer]
https://react-spectrum.adobe.com/releases/index.html

**Reply by bstro** [reply]
Not sure how I missed that! Thank you :)
